cmd/release-controller-api/http.go

```diff
@@ -153,12 +163,272 @@ func (c *Controller) userInterfaceHandler() http.Handler {
 	mux.HandleFunc("/api/v1/releasestreams/rejected", c.apiRejectedStreams)
 	mux.HandleFunc("/api/v1/releasestreams/all", c.apiAllStreams)
 
+	mux.HandleFunc("/api/v1/features/{release}/release/{tag}", c.apiFeatureReleaseInfo)
+	mux.HandleFunc("/features/{release}/release/{tag}", c.httpFeatureReleaseInfo)
```

Comment:
Have "Release" seems redundant to me...
Can these be `apiFeatureInfo` and `httpFeatureInfo` respectively?

Reply:
Methods renamed to `apiFeatureInfo` and `httpFeatureInfo` respectively.